### PR TITLE
[ISSUE #917] Add unit tests for EventMesh Runtime for Java

### PR DESCRIPTION
**Pull request by walterlife**
Fixes ISSUE https://github.com/apache/incubator-eventmesh/issues/917.

Motivation
Add missing unit tests

Modifications
Add unit tests for eventmesh-runtime module.

Documentation
- [ ] Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

**Review by qqeasonchen — COMMENTED** [review]
@walterlife you can add a new  more specific issue for this change

**Comment by walterlife** [comment]
> 

ok, add 917, and already modify pr description

**Comment by walterlife** [comment]
> @walterlife you can add a new more specific issue for this change

ok, add new issue: 917

**Comment by walterlife** [comment]
> @walterlife you can add a new more specific issue for this change

**Comment by walterlife** [comment]
> @walterlife you can add a new more specific issue for this change

@qqeasonchen ok, add new issue: 917

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/RemotingHelper.java [hunk 1]
```diff
@@ -50,6 +50,9 @@ public static String exceptionSimpleDesc(final Throwable e) {
 
     public static SocketAddress string2SocketAddress(final String addr) {
         int split = addr.lastIndexOf(":");
+        if (split == -1) {
```
Comment (ruanwenjun) [inline eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/RemotingHelper.java:53]: It's better to not do these kinds of changes in this PR, you just want to add UT right? Please revert the extra change.

File: build.gradle [hunk 2]
```diff
@@ -465,8 +465,8 @@ subprojects {
             dependency "com.github.stefanbirkner:system-rules:1.16.1"
             dependency "org.assertj:assertj-core:2.6.0"
 
-            dependency "org.mockito:mockito-core:2.23.0"
-            dependency "org.mockito:mockito-inline:3.6.28"
+            dependency "org.mockito:mockito-core:3.8.0"
+            dependency "org.mockito:mockito-inline:3.8.0"
```
Comment (ruanwenjun) [inline build.gradle:469]: Is there any reason we need to upgrade these version?

**Review by ruanwenjun — CHANGES_REQUESTED** [review]

File: build.gradle (hunk 2 above)
Comment (walterlife) [reply build.gradle:469]: In some unit-test scenarios, mock static methods are required, and this feature, mockito-inline, was introduced in higher versions, So upgrade the corresponding version.

File: eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/RemotingHelper.java (hunk 1 above)
Comment (walterlife) [reply eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/RemotingHelper.java]: ok, already revert.

File: build.gradle (hunk 2 above)
Comment (ruanwenjun) [reply build.gradle:469]: Ok, if so, we can remove the powermock.

**Review by ruanwenjun — DISMISSED** [review]
LGTM if the CI pass.

**Review by xwm1992 — APPROVED** [review]
LGTM

**Pull request merged**